docs/src/rsh/compute/index.md

```diff
@@ -618,9 +618,24 @@ band(a, b)    // &
 bior(a, b)    // |
 polyEq(a, b)  // ==, ===
 polyNeq(a, b) // !=, !==
+safeAdd(a, b)
+safeSub(a, b)
+safeMul(a, b)
+safeDiv(a, b)
+safeMod(a, b)
+safeMuldiv(a, b)
+veriAdd(a, b)
+veriSub(a, b)
+veriMul(a, b)
+veriDiv(a, b)
+veriMod(a, b)
+veriMuldiv(a, b)
```

Comment:
This is the wrong place to document this, because there's a separate section for `muldiv`